### DISCUSSION TITLE
Error create admin user

### DISCUSSION THREAD

**Discussion by nooyou** [Budibase/budibase#4240] [Help]
Hi,
I reinstalled Budibase with Docker and when I try to create the first user: the admin user it gives me an error:
![Sans titre](https://user-images.githubusercontent.com/18414238/151560183-acf63473-d5ef-47e9-acc2-65f9fc7fb0bf.jpg)
Where does this error come from?
Thanks.

**Comment by mjashanks** [comment]
If you want to start fresh (happy to lose the old version completely), I would try removing all your budibase docker containers - then reinstalling.

**Comment by nooyou** [comment]
Okay, I'll try.
Thank you.

**Comment by nooyou** [comment]
Yes thank you again
